webapp/src/main/java/org/apache/falcon/resource/SchedulableEntityManager.java

```diff
@@ -57,6 +59,91 @@ public APIResult getStatus(@Dimension("entityType") @PathParam("type") String ty
         }
     }
 
+    /**
+     * Delete the specified entity.
+     * @param request Servlet Request
+     * @param type Valid options are cluster, feed or process.
+     * @param entity Name of the entity.
+     * @param ignore colo is ignored
+     * @return Results of the delete operation.
+     */
+    @DELETE
+    @Path("delete/{type}/{entity}")
+    @Produces({MediaType.TEXT_XML, MediaType.TEXT_PLAIN, MediaType.APPLICATION_JSON})
+    @Monitored(event = "delete")
+    @Override
+    public APIResult delete(
+            @Context HttpServletRequest request, @Dimension("entityType") @PathParam("type") final String type,
+            @Dimension("entityName") @PathParam("entity") final String entity,
+            @Dimension("colo") @QueryParam("colo") String ignore) {
+        throw FalconWebException.newAPIException("delete on server is not" +
```

Comment:
Falcon can run in 2 modes: 
1. Distributed. In this case, Falcon prism is what user should use to perform CRUD operations on entities and direct CRUD operations against Falcon Server should not be allowed.
2. Embedded. In this case, there is no Prism server at all, there is just one instance of Server and all CRUD operations should be allowed against server in this mode.